<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color spaces</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="1">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Color space:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radRGB">
          <property name="text">
           <string>RGB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radHSV">
          <property name="text">
           <string>HSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radCIELab">
          <property name="text">
           <string>CIELab</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowFullSpace">
          <property name="text">
           <string>Show full space</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QVTKWidget" name="qvtkWidgetImage"/>
      </item>
      <item>
       <widget class="QVTKWidget" name="qvtkWidgetColorSpace"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_Grayscale_Image">
   <property name="text">
    <string>Open As Grayscale Image</string>
   </property>
  </action>
  <action name="actionOpen_RGBDI_Image">
   <property name="text">
    <string>Open As RGBDI Image</string>
   </property>
  </action>
  <action name="actionSaveSegmentation">
   <property name="text">
    <string>Save Segmentation</string>
   </property>
  </action>
  <action name="actionFlipImage">
   <property name="text">
    <string>Flip Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
